spec/sql_query_specs.jest.ts

```diff
@@ -644,3 +644,136 @@ describe("Query SELECT with special character in table", () => {
         expect(sql_query.replace(options, adhocFilters)).toBe(expQuery);
     });
 });
+
+/* check https://github.com/Altinity/clickhouse-grafana/issues/276 */
+describe("Query with Mustache templating", () => {
+    const query = "{{#METRIC_FIRST_BYTE}}\n" +
+        "    SELECT\n" +
+        "        toStartOfMinute(event_time) AS event_time,\n" +
+        "        service_name,\n" +
+        "        from_user,\n" +
+        "        count() as count\n" +
+        "    FROM $table\n" +
+        "\n" +
+        "    WHERE\n" +
+        "        $timeFilter\n" +
+        "        $conditionalTest(AND toLowerCase(service_name) IN ($metric),$metric)\n" +
+        "    GROUP BY\n" +
+        "        event_time,\n" +
+        "        from_user,\n" +
+        "        service_name\n" +
+        "{{/METRIC_FIRST_BYTE}}\n" +
+        "{{#METRIC_FIRST_PAINT}}\n" +
+        "    SELECT\n" +
+        "        toStartOfMinute(event_time) AS event_time,\n" +
+        "        service_name,\n" +
+        "        from_user,\n" +
+        "        count() as count\n" +
+        "    FROM $table\n" +
+        "\n" +
+        "    WHERE\n" +
+        "        $timeFilter\n" +
+        "        $conditionalTest(AND toLowerCase(service_name) IN ($metric),$metric)\n" +
+        "    GROUP BY\n" +
+        "        event_time,\n" +
+        "        from_user,\n" +
+        "        service_name\n" +
+        "{{/METRIC_FIRST_PAINT}}";
```

Comment:
METRIC_FIRST_PAINT and METRIC_FIRST_BYTE
looks the same
what exactly you try to do?

Reply:
@Slach  I will build more logic around this and I hope that this will bring more clarity. For sure it's very abstract at the moment.